source/scripts/Provider/EthereumProvider.ts

```diff
@@ -1,28 +1,49 @@
-export const EthereumProvider = () => {
-  const getNetwork = () => {};
-
-  const getAccounts = () => {};
-
-  const getChainId = () => {};
-
-  const getAddress = () => {};
+import { Web3Accounts } from '@syspollum/sysweb3-keyring';
+import {
+  getNftImage as web3NFTImage,
+  getTokenIconBySymbol as getTokenIcon,
+} from '@syspollum/sysweb3-utils';
+import { web3Provider as provider } from '@syspollum/sysweb3-network';
 
-  const getBalance = () => {};
-
-  const getXpub = () => {};
-
-  const getTokens = () => {};
-
-  const getState = () => {};
+export const EthereumProvider = () => {
+  const createAccount = () => Web3Accounts().createAccount();
+
+  const importAccount = (
+    mnemonicOrPrivateKey: string,
+    encryptedPwdAccount: string
+  ) => Web3Accounts().importAccount(mnemonicOrPrivateKey, encryptedPwdAccount);
+
+  const sendTransactions = (
+    fromPrivateKey: string,
+    toAddress: string,
+    value: number
+  ) => Web3Accounts().sendTransaction(fromPrivateKey, toAddress, value);
+
+  const getBalance = (walletAddress: string) =>
+    Web3Accounts().getBalance(walletAddress);
+
+  const changeNetwork = (networkId: number) =>
+    Web3Accounts().setActiveNetwork(networkId);
+
+  const getNFTImage = (NFTContractAddress: string, tokenId: number) =>
+    web3NFTImage(NFTContractAddress, tokenId);
+  const getUserNFT = (walletAddress: string) =>
```

Comment:
we can remove this and use the method available in @pollum-io/sysweb3-utils, as well as getTokenIconBySymbol and any other utility function that we can expose to the website directly from our lib